src/app/components/Buttons/GreyBtn.tsx

```diff
@@ -0,0 +1,9 @@
+function GreyBtn({ text }: { text: string }) {
+    return (
+        <>
+            <button className="rounded-none bg-adarkgrey m-auto p-3 font-medium text-lg text-awhite justify-center">{text}</button>
+        </>
```

Comment:
Ideally we would apply the "DRY" [Don't Repeat Yourself] principle as much as possible. 

Here we are creating three different components, but only changing one class name. [Unless I'm missing something]

A DRY way of doing it would be to have a single component and accept color as a prop.

Something like this maybe?

**Invocation**
```js
<Button backgroundColor="white">Call to action</Button>
```

**Definition**
```js
function Button({ children, backgroundColor }) {
    return (
        <>
            <button className=`rounded-none bg-a${backgroundColor} m-auto p-3 font-medium text-lg text-awhite justify-center`>{children}</button>
        </>
    )
}

export default Button;
```

If in the future we decide to change the font-size of the buttons, we would only have to change one component instead of three.